ui/src/app/shared/services/view-preferences-service.ts

```diff
@@ -86,7 +88,9 @@ const DEFAULT_PREFERENCES: ViewPreferences = {
         syncFilter: new Array<string>(),
         healthFilter: new Array<string>()
     },
-    pageSizes: {}
+    pageSizes: {},
+    showBanner: true,
```

Comment:
I think `showBanner` and `bannerContent` can be merged to `hideBannerContent: string`. If current bunner content and `hideBannerContent` are equal then banner should be hidden.